vendor/modules.txt

```diff
@@ -144,7 +144,7 @@ github.com/aws/smithy-go/waiter
 # github.com/benbjohnson/clock v1.3.5
 ## explicit; go 1.15
 github.com/benbjohnson/clock
-# github.com/conductorone/baton-sdk v0.2.34
+# github.com/conductorone/baton-sdk v0.2.43-0.20241022023644-64df71e8dd06
```

Comment:
both baton-jira and baton-jira-datacenter will need the release you make I guess then when your baton-sdk PR is merged